static/app/components/events/eventStatisticalDetector/aggregateSpanDiff.tsx

```diff
@@ -85,17 +93,86 @@ function AggregateSpanDiff({event, project}: AggregateSpanDiffProps) {
     relativeDays: 7,
     retentionDays: 30,
   });
-  const {data, isLoading, isError} = useFetchAdvancedAnalysis({
+
+  const {
+    data: rcaData,
+    isLoading: isRcaLoading,
+    isError: isRcaError,
+  } = useFetchAdvancedAnalysis({
     transaction,
     start: (start as Date).toISOString(),
     end: (end as Date).toISOString(),
     breakpoint: breakpointTimestamp,
     projectId: project.id,
+    enabled: !isSpansOnly,
+  });
+
+  const {
+    data: spansData,
+    isLoading: isSpansDataLoading,
+    isError: isSpansDataError,
+  } = useSpanMetrics({
+    search: new MutableSearch(`transaction:${transaction} has:span.description`),
+    fields: [
+      'span.op',
+      'span.description',
```

Comment:
This group by is going to be expensive since the descriptions themselves arent normalized, i think you'll want something like `any(span.description)`